### ISSUE TITLE
[feature request] Keyboard shortcuts

### ISSUE THREAD

**Issue by RedirectCleaner** [hackademix/noscript#45]
It would be nice to get the possibility to set keyboard shortcuts for "temporarily allow all scripts" and "revoke all temporarily permissions"

**Comment by apiraino** [comment]
IIRC this was available before 10.x and it was really useful

**Comment by schrodog** [comment]
It would be nice to add one more shortcut for "set current website as untrusted"

**Comment by iansinnott** [comment]
For anyone else who runs into this limitation, there's a little-known feature in Firefox that might help. Firefox allows you to enable keyboard shortcuts for various extensions in the addons settings.

![2020-08-09 at 2 59 PM](https://user-images.githubusercontent.com/3154865/89726803-43a03380-da51-11ea-8315-e30f147ba986.png)

**Comment by exquo** [comment]
In the "Extension Shortcuts" screen in`about:addons` that @iansinnott has mentioned there is a "Toggle permissions" field, but setting it to any key combo does not seem to have any effect. Changing the "NoScript UI" key shortcut _does_ work though.

**Comment by nickdevereaux** [comment]
https://github.com/hackademix/noscript/blob/984e68e3cf2dc3c9446b608484c4c1bab6f544a6/src/bg/main.js#L110

It looks like it's just stubbed?

**Comment by hackademix** [comment]
> https://github.com/hackademix/noscript/blob/984e68e3cf2dc3c9446b608484c4c1bab6f544a6/src/bg/main.js#L110
> 
> It looks like it's just stubbed?

Yes it is, because under the new permissions it wasn't obvious what "Toggle permissions for this site" would mean, and how useful it would be.

Maybe I should rename it in "Toggle restrictions for this tab" and turn it into a shortcut to the "Disable/enable restrictions for this tab" feature?
